xcell/mappers/mapper_KiDS1000.py

```diff
@@ -79,9 +78,8 @@ def _load_catalog(self):
                 cat_out = cat
         return cat_out
 
-    def _set_mode(self, mode=None):
-        if mode is None:
-            mode = self.mode
+    def _set_mode(self):
```

Comment:
Why do you remove the `mode` argument? (or why do you remove it here and keep it in KV450?)

Reply:
The logic is that the mode is set in the yml file, and cannot be changed afterwards

Reply:
Make sense (and I like this more)